examples/react/css/src/toggle-refinement.css

```diff
@@ -0,0 +1,47 @@
+.ais-ToggleRefinement-label {
+  font-size: 0.875rem;
+  line-height: 1.75rem;
+  display: block;
+  cursor: pointer;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.ais-ToggleRefinement-checkbox {
+  appearance: none;
+  height: 18px;
+  pointer-events: none;
+  width: 32px;
+  position: relative;
+  border: 1px solid rgba(var(--ais-muted-color-rgb), 0.3);
+  background-color: rgb(var(--ais-background-color-rgb));
+  border-radius: 9999px;
+}
+
+.ais-ToggleRefinement-checkbox:checked {
+  background-image: none !important;
```

Comment:
Are you doing this because of this rule in `common.css`?
```css
input[class^='ais-'][type='checkbox']:checked { /* ... */ }
```
I think we can update the selector and move it to `refinement-list.css` since your checkbox is more like a switch.